### PR TITLE
Here we go again

### PR DESCRIPTION
**Pull request by luciusmagn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,14 @@ Testausserverin siistein botti
 
 ## Ominaisuuksia
 
-### Haskell ajuri
-Kun pingaat kamelin viestiin, jossa on Haskell-koodipalikka
+### Kieliajurit
+Kun pingaat kamelin viestiin, jossa on tuettukieli-koodipalikka
```
Comment (lajp) [inline README.md:9]: Im pretty sure "tuettukieli" is not a word. Just put something like "jossa on tuetun kielen koodipalikka"

**Review by lajp — CHANGES_REQUESTED** [review]
Fix Finnish, otherwise seems fine so far

File: README.md (hunk 1 above)
Comment (luciusmagn) [reply README.md]: fixed

**Review by lajp — APPROVED** [review]
LGTM!

**Pull request merged**